File: servers/hear/src/main/scala/com/arkr/hear/config/ApplicationConfig.scala
package com.arkr.hear.config

import com.arkr.hear.SpringComponentScanMarker
import com.arkr.hene.data.config.DataConfig
import com.arkr.hene.mq.kafka.KafKaConfig
import org.springframework.context.annotation.{ComponentScan, Configuration, Import}

/**
  * Created by hztanhuayou on 2017/8/26
  */
@Configuration
@Import(value = Array(
  classOf[DataConfig],
  classOf[KafKaConfig]
))
@ComponentScan(basePackageClasses = Array(classOf[SpringComponentScanMarker]))
class ApplicationConfig {
  // do nothing
}

File: provider/echo/src/main/java/com/arkr/hexe/ApplicationConfig.java
package com.arkr.hexe;

import com.arkr.boot.data.config.DataConfig;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

/**
 * Created by hztanhuayou on 2017/8/27
 */
@Configuration
@Import(DataConfig.class)
public class ApplicationConfig {
}
